test-clients/src/main/resource/contract/contracts/AtomicCryptoTransfer/AtomicCryptoTransfer.json

```diff
@@ -0,0 +1,492 @@
+[
```

Comment:
It is a damn shame that JSON doesn't allow for comments.  (JSON5 did, but never caught on. TOML/YAML _do_ allow for comments.). Because what the heck is this file _for_?  Which test or configuration _uses it_?  Which of the values below are just placeholders that could be anything, and which are _critical_ for the proper functioning of the test?   Dunno.  Perhaps, though, a readme file in this directory would help?  Or put it somewhere else, maybe in the actual test that _uses_ this 500 line JSON blob?

Reply:
This is one of the outputs of the solc compiler which (among other things) generates something called the abi (application binary interface).  This file is not really meant for human consumption although you can make sense of it if you try.  You'll notice that all other compiled contracts have a similar files in their appropriate sub-folder.   A README at the top level could be useful but most people working with compiled solidity should immediately recognize what these files are.

Reply:
Thanks, IDNKT!